src/styles/blocks/card.scss

```diff
@@ -0,0 +1,97 @@
+.product-card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: $card-width;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  padding: 0 16px 16px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    margin: 32px auto 40px;
+  }
+
+  &__name {
+    display: inline-block;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $text-main-color;
+    margin-bottom: 4px;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-additional-colour;
+    margin-bottom: 16px;
+  }
+
+  &__review {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+
+  &__number {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-main-color;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    line-height: 18px;
+    margin-bottom: 16px;
+  }
+
+  &-word {
+    font-weight: 400;
```

Comment:
Default value.
This selector must be nested to **&__price** or written as follows **&__price-word**

Reply:
Fit it